python_modules/dagster/dagster_tests/general_tests/grpc_tests/test_get_current_runs.py

```diff
@@ -0,0 +1,50 @@
+import subprocess
+import sys
+
+from dagster._api.list_repositories import sync_list_repositories_grpc
+from dagster._grpc.client import DagsterGrpcClient
+from dagster._grpc.server import wait_for_grpc_server
+from dagster._utils import file_relative_path, find_free_port
+
+
+def test_load_grpc_server(capfd):
```

Comment:
little confused what's going on with this test since it doesn't call get_current_runs - included intentinoally?